get_anndata.py

```diff
@@ -144,6 +144,8 @@ def make_pheno_cov(
         sample_covs_cells_df['sex'] = sample_covs_cells_df['sex'].fillna(0)
     if fill_in_age:
         sample_covs_cells_df['age'] = sample_covs_cells_df['age'].fillna(mean_age)
+# drop rows with missing values (SAIGE throws an error otherwise:  https://batch.hail.populationgenomics.org.au/batches/435978/jobs/91)
```

Comment:
```suggestion
   # drop rows with missing values (SAIGE throws an error otherwise:  https://batch.hail.populationgenomics.org.au/batches/435978/jobs/91)
```